src/flightlog_index.js

```diff
@@ -140,9 +142,8 @@ export function FlightLogIndex(logData) {
           switch (frameType) {
             case "P":
             case "I":
-              var frameTime =
-                frame[FlightLogParser.prototype.FLIGHT_LOG_FIELD_INDEX_TIME];
-
+              let frameTime = frame[FlightLogParser.prototype.FLIGHT_LOG_FIELD_INDEX_TIME];
```

Comment:
To fix the sonar issue either use { } in this case or simply declare `frameTime` above the switch statement and remove let from this line. Perhaps this is a better place for `lastTime` too.

Reply:
The sonar is relaxed.